shared/schemas/workflow-options-output-schema.json

```diff
@@ -0,0 +1,37 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Workflow Results",
+    "description": "Results of a workflow execution",
+    "type": "object",
+    "properties": {
+        "completedWith": {
+            "description": "The state of workflow completion.",
+            "type": "string",
+            "enum": [
+                "error",
+                "success"
+            ]
+        },
+        "description": {
+            "description": "High-level summary of the current status, free-form text, human readable.",
+            "type": "string"
+        },
+        "nextWorkflows": {
+            "description": "List of workflows suggested to run next. Items at lower indexes are of higher priority.",
+            "type": "array",
+            "items": {
+                "$ref": "workflow-ref-schema.json"
+            }
+        },
+        "options": {
+            "description": "Additional structured output of workflow processing. This can contain identifiers of created resources, links to resources, logs or other output.",
+            "type": "array",
+            "items": {
+                "$ref": "workflow-option-schema.json"
+            }
+        }
+    },
+    "required": [
+        "completedWith"
```

Comment:
My concern about this if this schema is going to be used while a workflow is still running or only at the end of it.
Since while the workflow is running, it may produce result items that we can/should render in the UI.
However, since the workflow is still running, there is no value for `completedWith` at that time.
@batzionb how do we fetch the workflow results ATM, while the workflow is still running?

Reply:
Per comment above, the `completedWith` will be removed.

Reply:
> My concern about this if this schema is going to be used while a workflow is still running or only at the end of it.
> Since while the workflow is running, it may produce result items that we can/should render in the UI.
> However, since the workflow is still running, there is no value for completedWith at that time.
> @batzionb how do we fetch the workflow results ATM, while the workflow is still running?

This is a good point. I see it more as a UX question. While workflow is running, the results card should have some kind of busy state or not be shown at all. Not sure how the API should be effected by this if at all.